code/game/objects/items/storage/boxes.dm

```diff
@@ -289,6 +279,16 @@
 	for(var/i in 1 to 7)
 		new /obj/item/grenade/flashbang(src)
 
+/obj/item/storage/box/stingbangs

```

Comment:
undocumented change?

Reply:
No, me being dumb. Should be taken care of now!